todo_app/test/test_view_model.py

```diff
@@ -14,7 +14,7 @@ def test_shows_todo_items():
         Card(3, "Done", "Third item", "2020-10-27")
     ]
 
-    view_model = ViewModel(items)
+    view_model = ViewModel(items,False)
```

Comment:
Minor, but this could be a nice occasion when you could have used a default value for the argument, so that you don't need to update everywhere that creates a ViewModel